arm-network/2017-06-01/swagger/networkInterface.json

```diff
@@ -587,6 +598,20 @@
           "type": "string",
           "description": "The destination port or range."
         },
+        "sourcePortRanges": {
```

Comment:
From description it looks like a settable property `sourcePortRanges` & `destinationPortRanges` are being added to the response model `EffectiveNetworkSecurityRule`. Could you please point me the API that can set these values to better understand the scenario?